### PR TITLE
Fixed themeName not found error - Jilliane Vina

### PR DESCRIPTION
**Pull request by j1000-dev**

### CODE REVIEW

File: src/pages/App/index.jsx
Comment (ZedTT) [inline src/pages/App/index.jsx]: Do we want console logs in master?

File: src/pages/App/index.jsx
Comment (ZedTT) [inline src/pages/App/index.jsx]: extra newline not that it matters

File: src/pages/App/index.jsx
Comment (ZedTT) [inline src/pages/App/index.jsx]: Do we want console logs in master?

**Review by ZedTT — DISMISSED** [review]
Looks good except that we maybe shouldn't have console logs in mater.

File: src/pages/App/index.jsx
Comment (ghost) [reply src/pages/App/index.jsx]: ya, it doesnot matter

File: src/pages/App/index.jsx
Comment (ghost) [reply src/pages/App/index.jsx]: i think she just test

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ZedTT — APPROVED** [review]
LGTM after removing the extra logs. Note that I haven't tested that it runs, and only reviewed the code through the browser. I believe someone else from your team has reviewed this, though.

Let me know if you would prefer that I test the environment before merging, otherwise feel free to merge.

**Pull request merged**
